lisa/tools/sar.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from decimal import Decimal
+from typing import List, cast
+
+from lisa.executable import Tool
+from lisa.messages import NetworkPPSPerformanceMessage
+from lisa.operating_system import Posix
+from lisa.util.process import ExecutableResult, Process
+
+
+class Sar(Tool):
+    # 06:37:41        IFACE   rxpck/s   txpck/s    rxkB/s    txkB/s   rxcmp/s   txcmp/s  rxmcst/s   %ifutil # noqa: E501
+    # 06:37:42           lo      0.00      0.00      0.00      0.00      0.00      0.00      0.00      0.00 # noqa: E501
+    # 06:37:42         eth0   2856.00   2857.00    186.86    187.28      0.00      0.00      0.00      0.00 # noqa: E501
+    #
+    # 06:37:42        IFACE   rxpck/s   txpck/s    rxkB/s    txkB/s   rxcmp/s   txcmp/s  rxmcst/s   %ifutil # noqa: E501
+    # 06:37:43           lo      0.00      0.00      0.00      0.00      0.00      0.00      0.00      0.00 # noqa: E501
+    # 06:37:43         eth0   3195.00   3194.00    209.04    209.33      0.00      0.00      0.00      0.00 # noqa: E501
+    sar_results_pattern = re.compile(r"(IFACE[\w\W]*?)(?=IFACE|\Z)", re.MULTILINE)
+
+    @property
+    def command(self) -> str:
+        return "sar"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        posix_os: Posix = cast(Posix, self.node.os)
+        posix_os.install_packages("sysstat")
+        return self._check_exists()
+
+    def get_statistics_async(
+        self, key_word: str = "DEV", interval: int = 1, count: int = 300
```

Comment:
Can it be shorter? 5 minutes is too long to get the statistics.

Reply:
I don't think perf testing can get a stable result in short time. but I will have a try.

Reply:
updated into 120